python/src/com/jetbrains/python/psi/impl/PythonLanguageLevelPusher.java

```diff
@@ -65,11 +67,16 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+import static com.jetbrains.python.psi.LanguageLevel.*;
+
 /**
  * @author yole
  */
 public class PythonLanguageLevelPusher implements FilePropertyPusher<LanguageLevel> {
   public static final Key<LanguageLevel> PYTHON_LANGUAGE_LEVEL = Key.create("PYTHON_LANGUAGE_LEVEL");
+  public static final Set<Pair<Integer, Integer>> COMPATIBLE_LEVELS = Sets.newHashSet(
+    Pair.create(PYTHON26.ordinal(), PYTHON27.ordinal()),
+    Pair.create(PYTHON33.ordinal(), PYTHON34.ordinal()));
```

Comment:
Consider using `EnumMap`

Reply:
It's really would be more of a multimap since one language level can be compatible with several others. Unless I'm misunderstanding something?

Reply:
The type could be `EnumMap<LanguageLevel, EnumSet<LanguageLevel>>`